client/app.jsx

```diff
@@ -77,6 +77,16 @@ export default function App(props) {
     setSearchFormOpen(false);
   };
 
+  const signOut = () => {
+    fetch('/api/sign-out')
+      .then(res => res.json())
+      .then(user => {
+        setUser(null);
+        window.location.reload();
```

Comment:
Is reloading the page really necessary? seems hacky. ideally you can just reset the state of your app/component(s)